lib/utils.py

```diff
@@ -44,3 +47,13 @@ def stack_images(images):
         images,
         axes=numpy.concatenate(new_axes)
         ).reshape(new_shape)
+
+
+class FullHelpArgumentParser(argparse.ArgumentParser):
+    """Identical to the built-in argument parser, but on error
+    it prints full help message instead of just usage information
+    """
+    def error(self, message):
+        self.print_help(sys.stderr)

```

Comment:
kinda hacky solution, maybe we don't need to print full help in case of argument parsing error?